methods_and_recursion/bottles_of_beer.rb

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+
+def pluralize(number, string)
+  string += 's' unless number == 1
+  return string
```

Comment:
What if this method returned the number and the string, i.e. "99 bottles" or "No more bottles"